### ISSUE TITLE
cp2k

### ISSUE THREAD

**Issue by jautschbach** [ubccr/software-layer#118]
Please build a recent version of the CP2K package (https://www.cp2k.org/)
See also CCR support ticket 34143
The latest version is 2023.2 (https://www.cp2k.org/version_history); please install that one if possible.

**Comment by tonykew** [comment]
Test build complete - awaiting a test set from Jochen to verify before publishing.

**Comment by tonykew** [comment]
Published version 2023.1 to the CCR software stack, which seems (empirically)
to be more stable than 2023.2

```
$ module spider cp2k

-------------------------------------------------------------------------------------------
  cp2k: cp2k/2023.1
-------------------------------------------------------------------------------------------
    Description:
[...]
You will need to load all module(s) on any one of the lines below before the "cp2k/2023.1" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
$ module load gcc/11.2.0 openmpi/4.1.1 cp2k/2023.1
$ cp2k.popt --help

 cp2k.popt [-c|--check] [-e|--echo] [-h|--help]
           [-i] <input_file>
           [-mpi-mapping|--mpi-mapping] <method>
           [-o] <output_file>
           [-r|-run] [-s|--shell] [--xml]
[...]
$ cp2k.psmp --help

 cp2k.psmp [-c|--check] [-e|--echo] [-h|--help]
           [-i] <input_file>
           [-mpi-mapping|--mpi-mapping] <method>
           [-o] <output_file>
           [-r|-run] [-s|--shell] [--xml]
[...]
$ 
```
